src/adn30.f

```diff
@@ -26,7 +25,6 @@ FUNCTION ADN30(IDN,L30)
 
 C----------------------------------------------------------------------
 C----------------------------------------------------------------------
-
       IF(LEN(ADN30).LT.L30         ) GOTO 900
```

Comment:
What is the significance of the number 30 to this function?

The number 30 is used in the name of the function and in some variable names, but what does this function have to do with 30?

Reply:
That's a really good question!

@jack-woollen, does that number have some sort of legacy significance back from the early days of library development?  If not, and if there's no other reason to keep it, then maybe we should remove it from this and the idn30 routines to prevent further confusion.

Reply:
Well we can't remove the 30 without changing the function names, which is backwards incompatible.

But we could document the reason for 30.

Reply:
No application code should be calling these functions directly, so the only issue should be just making all of the changes internal to the library itself and within our test_bort code so that it's internally compatible.

We should still document this change in the release notes, but I don't really see backwards-incompatibility as an issue, since no external codes should be directly referencing either of these functions.

But of course I would like to hear from Jack in case there's some reason for the 30 that we're not aware of?